appveyor.yml

```diff
@@ -38,10 +38,11 @@ build_script:
 test_script:
   - yarn test:setup
   - ./script/test-appveyor.bat
```

Comment:
This wrapper script sets the `APPVEYOR_TEST_RESULT` environment variable (for reasons I can't recall) so that Appveyor will behave like the other platforms rather than continue on to the next step of the script in the event of a non-zero exit code.